test/sql/test_function/R/test_named_argments

```diff
@@ -0,0 +1,44 @@
+-- name: test named arguments for table functions
+select * from TABLE(generate_series(start=>2, end=>5));
+-- result:
+2
+3
+4
+5
+-- !result
+select * from TABLE(generate_series(start=>2, stop=>5));
+-- result:
+E: (1064, "Getting analyzing error. Detail message: Unknown table function 'generate_series(start=>2,stop=>5)', the function doesn't support named arguments or has invalid arguments.")
```

Comment:
step is an optional argument, why it is not supported?

Reply:
the second should be `end`, not `stop`, invalid arguments.